docs/src/docs_best_practices/how-to/requirements_checklist.md

```diff
@@ -0,0 +1,38 @@
+# Complete Basic Requirements Checklist
+
+Sienna packages should follow the [Diataxis](https://diataxis.fr/)
+framework, be strictly compiled with [`Documenter.jl`](https://documenter.juliadocs.org/stable/)
+v1.0 or greater, and be automatically formatted with
+[`JuliaFormatter.jl`](https://domluna.github.io/JuliaFormatter.jl/stable/).
+
+## For New Packages
+
+The [`SiennaTemplate.jl`](https://github.com/NREL-Sienna/SiennaTemplate.jl) Git repo has the
+required environments and formatting and documentation code. Start from this template.
+
+## For Existing Packages
+
+Existing Sienna packages will need to be updated with these requirements, but these will
+only need to be addressed once:
+
+ 1. Organize the top-level documentation to follow the [Diataxis](https://diataxis.fr/)
+    framework (plus a welcome page/section). This might be a significant undertaking. See:
+    
+      + How to [Write a How-to Guide](@ref)
+      + How to [Write a Tutorial](@ref)
+      + How to [Organize APIs and Write Docstrings](@ref)
+
+ 2. Replace `compat` requirements of `Documenter = "0.27"` in `docs/` environment with
```

Comment:
It could be helpful to specify the Project.toml file in the docs/ folder is what needs to be changed